webapp/channels/src/components/preparing_workspace/preparing_workspace.tsx

```diff
@@ -225,16 +227,15 @@ const PreparingWorkspace = (props: Props) => {
         const sendFormStart = Date.now();
         setSubmissionState(SubmissionStates.Submitting);
 
-        if (form.organization && !isSelfHosted) {
+        if (!form.teamMembers.skipped && !isConfigSiteUrlDefault && !isSelfHosted) {
```

Comment:
I guess it doesn't hurt this additional check but in cloud, the SIteURL is always configured to their workspace url at this point in the onboarding.